src/sentry/api/bases/project.py

```diff
@@ -59,6 +59,7 @@ class ProjectEndpoint(Endpoint):
     permission_classes = (ProjectPermission,)
 
     def convert_args(self, request, organization_slug, project_slug, *args, **kwargs):
+        self.bail_on_xorg(request, slug=organization_slug)
```

Comment:
one concern I have is we're not doing a lot of extra work to query the database for permissions, re-build access hooks, etc